.github/workflows/supabase-types.yml

```diff
@@ -0,0 +1,47 @@
+name: Update database types
+
+on:
+  push:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    env:
+      PROJECT_ID: "lelwqeowubwpkhjlzsiu"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      - uses: actions/setup-node@v2.1.5
+        with:
+          node-version: 20.10.0
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Generate Supabase types
+        run: npm run update-types
+
+      - name: check for file changes
+        id: git_status
+        run: |
+          echo "status=$(git status -s)" >> $GITHUB_OUTPUT
+
+      - name: Commit files
+        if: ${{contains(steps.git_status.outputs.status, ' ')}}
+        run: |
+          git config --global user.name 'github-actions'
+          git config --global user.email 'github-actions@github.com'
+          git add supabase/types.ts
+          git commit -m 'chore: update Supabase types'
+          git push
+
+      - name: Push changes
```

Comment:
You use this step but the latest command of the previous step seems to be `git push`?